src/libraries/Microsoft.Extensions.DependencyInjection.Specification.Tests/src/Microsoft.Extensions.DependencyInjection.Specification.Tests.ref.cs

```diff
@@ -0,0 +1,459 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is this necessary? Technically this isn't "API" that we document. They are tests that we ship so DI implementers can ensure their implementations are correct.

Reply:
If DI implementers depend on these tests than those tests are actually API, aren't they? :)

Reply:
I would opt this one out. It is already opted out in the current code.